setup.py

```diff
@@ -24,7 +24,7 @@
     "nibabel>=2.4.0",
     "numpy",
     "seaborn",
-    "scikit-image>=0.16.0",
+    "scikit-image>=0.19.1",
```

Comment:
`skimage.draw.disk` should be available starting from `scikit-image` version `0.17.x`, see [here](https://scikit-image.org/docs/0.17.x/api/skimage.draw.html#skimage.draw.disk).

And if we require only `scikit-image>=0.17.1`, then we can keep the support `Python 3.6`, since `scikit-imate` version `0.17.1` [had wheels](https://pypi.org/project/scikit-image/0.17.1/#files) for this Python version.

Reply:
Thank you for the comment @FrancescoCasalegno, I saw this change in the log of `0.19.0` so I thought it was coming with this version! I am going to do the changes.

Reply:
Done in 7413e2a